### ISSUE TITLE
ultralytics v11 segment

### ISSUE THREAD

**Issue by VohminV** [airockchip/rknn-toolkit2#333]
```0: 320x320 (no detections), 20.3ms
Speed: 0.8ms preprocess, 20.3ms inference, 0.6ms postprocess per image at shape (1, 3, 320, 320)
Masks present: False`

` results = model(cropped_frame, conf=0.8, imgsz=320)

        detection_found = False

        for result in results:
            masks = result.masks
            boxes = result.boxes

            print("Masks present:", masks is not None and len(masks.data) > 0) 


The mass disappeared after the model was converted to rknn model

**Comment by VohminV** [comment]
`import os
from ultralytics import YOLO
from IPython.display import display, Image
from IPython import display
import torch
import cv2
import matplotlib.pyplot as plt
import numpy as np

display.clear_output()


model = YOLO("yolo11n.pt")

model.train(data="/kaggle/input/helipad/data.yaml", epochs=150, imgsz=320)

metrics = model.val()`
